install_files/ansible-base/roles/build-securedrop-app-code-deb-pkg/files/copy_app_code_files.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python
+
+import glob
+import os
+import shutil
+import subprocess
+import sys
+
+SECUREDROP_DIR = sys.argv[1]
+BUILD_DIR = sys.argv[2]
+GLOBS = {'.': ['*.py', 'COPYING', 'wordlist'],
+         'dictionaries': ['*.txt'],
+         'journalist_templates': ['*.html'],
+         'management': ['*.py'],
+         'requirements': ['securedrop-requirements.txt'],
+         'source_templates': ['*.html'],
+         'static/js': ['*.js'],
+         'static/js/libs': ['*.js']}
+PATHS = ['static/fonts', 'static/i']
+
+os.makedirs(BUILD_DIR)
+
+for rel_path, glob_list in GLOBS.iteritems():
+    try:
+        os.makedirs(os.path.join(BUILD_DIR, rel_path))
```

Comment:
I think you want `exist_ok=True` here instead of the `try/except`, though that is a `python3` option. Since this is a build script and not the main app, it might be worth it to update to `python3` to have one less thing to change in the future.